agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -116,14 +116,37 @@ func newPgStatMonitorQAN(q *reform.Querier, dbCloser io.Closer, agentID string,
 	}, nil
 }
 
-func getPGMonitorVersion(q *reform.Querier) (pgMonitorVersion float64, err error) {
+func getPGMonitorVersion(q *reform.Querier) (float64, error) {
 	var v string
-	err = q.QueryRow(fmt.Sprintf("SELECT /* %s */ pg_stat_monitor_version()", queryTag)).Scan(&v)
+	err := q.QueryRow(fmt.Sprintf("SELECT /* %s */ pg_stat_monitor_version()", queryTag)).Scan(&v)
 	if err != nil {
-		return
+		return 0, err
+	}
+
+	return parsePGMonitorVersion(v)
+}
+
+func parsePGMonitorVersion(version string) (float64, error) {
+	split := strings.Split(version, ".")
+	switch len(split) {
+	case 1:
+		return strconv.ParseFloat(split[0], 64)
+	case 2:
```

Comment:
did you ask pg_stat_monitor team if it's possible to have less then 3 numbers in version?

Reply:
No, I will ping them for answer.